src/definitions/modules/dropdown.js

```diff
@@ -2075,7 +2075,7 @@ $.fn.dropdown = function(parameters) {
                     return;
                   }
                   if(isMultiple) {
-                    if($.inArray(module.escape.htmlEntities(String(optionValue)), value) !== -1) {
+                    if($.inArray(module.escape.htmlEntities(String(optionValue)), String(value)) !== -1) {
```

Comment:
You are casting an array (`value` is an array here!) to a string. This results in wrong results if the value has more than one item
```console
String([1,2,3,4]) = "1,2,3,4"
$.inArray("2","1,2,3,4") -> 2 -> wrong
```
You have to cast each single possible item in the array instead

```javascript
if($.inArray(module.escape.htmlEntities(String(optionValue)), value.map(function(v){return String(v);})) !== -1) {
```

Reply:
Idk how I ended up casting this way, got it working so I guess it was fine for me,
anyway, fixed the fix, and got my branch cleaned

Thanks for the review though